src/ecosystem/corals/Corals.jl

```diff
@@ -119,9 +119,7 @@ Generate colony area data based on Bozec et al., [1].
        management in the Whitsundays.
      https://doi.org/10.13140/RG.2.2.26976.20482
 """
-function colony_areas()
-    # The coral colony diameter bin edges (cm) are: 0, 2, 5, 10, 20, 40, 80
-    edges = bin_edges(; unit=:cm)
+function colony_areas(edges)
```

Comment:
Same comment here. This will break any code that relies on the previous method.

To be clear, anyone who had written `ADRIA.colony_areas()` will see their code break.